dvc/repo/imp_url.py

```diff
@@ -27,6 +27,7 @@ def imp_url(self, url, out=None, resume=False, fname=None, erepo=None):
     self.check_dag(self.stages() + [stage])
 
     with self.state:
+        stage.remove_outs(ignore_remove=False, force=False)
```

Comment:
it should probably behave the same way as a regular run. Ask for confirmation? Especially if data is not cached or saved to remote at least

Reply:
Ok